src/gophernicus.c

```diff
@@ -34,11 +34,12 @@
  * Libwrap needs these defined
  */
 #ifdef HAVE_LIBWRAP
+#include <syslog.h>
```

Comment:
AFAICT `syslog` dosen't use libwrap..

Hide this behind a compile-time option, something like

```
#ifndef NO_SYSLOG
#include <syslog.h>
#endif
```

For the time being, and then in the new build system I will detect syslog.h.

Reply:
`syslog` doesn't use `libwrap`, but we're using `syslog` symbols (`LOG_DEBUG` and `LOG_ERR`) for `libwrap`.
The current master already assumes `syslog.h` exists, therefore I prefer not to define a new feature macro in this commit.

 I included `syslog.h` here to pull implementation details out of `gophernicus.h`. `log.c` and `gophernicus.c` need to include `syslog.h`; the other translation units don't need this header anymore, since custom logging functions are provided by `gophernicus.h`.

Reply:
Right, I see.